### PR TITLE
Update codegen version

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:
UpdateApplicationDescription is removed from service.pb.auth.proto by this PR because this is not implemented.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by knanao** [comment]
@nghialv 
Please check the permission of github actions because we cannot run codegen job like this.
https://github.com/pipe-cd/pipecd/actions/runs/2318686040

**Comment by nghialv** [comment]
@knanao It was because we have a list of actions that could be executed.
I have just added the new one to that whitelist, so please re-run the workflow.

**Comment by knanao** [comment]
Thank you! Everything is passed. :D

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -50,7 +50,7 @@ jobs:
 
   chart:
     name: Helm Chart
-    runs-on: ubuntu-latest      
+    runs-on: ubuntu-latest
```
Comment (nghialv) [inline .github/workflows/build.yaml:53]:
```suggestion
    runs-on: ubuntu-latest
```

**Review by nghialv — APPROVED** [review]
Great!

**Review by khanhtc1202 — APPROVED** [review]
Nice work 🙌

**Pull request merged**